packages/front-end/components/HomePage/ExperimentsGetStarted.tsx

```diff
@@ -1,8 +1,7 @@
-import React from "react";
+import React, { useMemo, useState } from "react";
```

Comment:
Is there a way to make eslint auto-fix this?  I noticed vs code does this a lot with react auto-imports.

Reply:
It looks like there's an ESLint plugin related to imports: https://www.npmjs.com/package/eslint-plugin-import

It looks like it would fix things.

I think it might make sense to add this in a separate PR because it's likely to create a huge diff.

I've created a ticket for it as I think it warrants doing separate from other tasks: https://github.com/growthbook/growthbook/issues/521